docs/cli.md

```diff
@@ -272,3 +217,58 @@ You must unbind delete all bindings before deprovisioning an instance.
 $ svcat deprovision ups-instance
 deleted ups-instance
 ```
+
+## Deregister a service broker with the cluster
+Deregistering is the process of removing a broker and it's associated classes and plans from the cluster.
+You must delete all active instances of it's classes before deregistering a broker.
+```console
+$ svcat deregister ups-broker --scope cluster
+Successfully removed broker "ups-broker"
+```
+
+# Namespaced Resource Support
+
+svcat supports interaction with the namespaced versions of Service Catalog resources. The `scope` flag is
+used to indicate whether to execute the command in the `cluster`, `namespace`, or `all` scope. 
+
+## Register a Namespaced Broker
+
+svcat registers/deregisters brokers as namespaced brokers by default. To register a broker as a cluster-wide broker, use the
+`--scope` flag.
+```console
+$ svcat register ups-broker --url http://ups-broker-ups-broker.ups-broker.svc.cluster.local --scope cluster
```

Comment:
Shouldn't this command omit the `--scope` flag because we are talking about how to install a namespaced broker?